src/java.base/share/classes/java/lang/VirtualThread.java

```diff
@@ -1139,6 +1139,7 @@ Thread.State threadState() {
             case WAITING:
             case TIMED_WAITING:
             case YIELDING:
+            case BLOCKING:
```

Comment:
These cases are somewhat sorted (by pair X and TIMED_X); please keep them that way by putting `BLOCKING` before `PARKING`.